src/evict/evict_inline.h

```diff
@@ -588,30 +588,44 @@ __wti_evict_hs_dirty(WT_SESSION_IMPL *session)
       ((uint64_t)(conn->evict->eviction_dirty_trigger * bytes_max) / 100));
 }
 
+/*
+ * __evict_check_user_ok_with_eviction --
+ *     Check if the user wants to abort eviction in this session. #private
```

Comment:
Is it better to use `#private` as part of a seperate PR. Where we do it for every function at once? It may cause inconsistencies. As other function won't have it yet causing ambiguity.